iohk/NixOps.hs

```diff
@@ -73,6 +75,9 @@ projectURL     IOHK            = "https://github.com/input-output-hk/iohk-nixops
 projectURL     Nixpkgs         = "https://github.com/nixos/nixpkgs.git"
 projectURL     Stack2nix       = "https://github.com/input-output-hk/stack2nix.git"
 
+nixpkgsRevTarball :: Commit -> URL
+nixpkgsRevTarball (Commit rev) = URL $ "https://github.com/NixOS/nixpkgs/archive/"<>rev<>".tar.gz"
```

Comment:
If I'm reading code correctly, it takes owner/repo from `nixpkgs-src.json` and and then writes it back.

Currently, if someone would change the owner to `input-output-hk` and bump the revision, it would read the json file, then fetch from hardcoded URL and fail since commit wouldn't be in `NixOS` repo.

I'm asking if we can use the information read from the JSON as truth and then only change the revision as per CLI options.